src/api/treeTrackerApi.js

```diff
@@ -68,51 +71,75 @@ export default {
       handleError(error);
     }
   },
-  approveCaptureImage(id, morphology, age, captureApprovalTag, speciesId) {
+  approveCaptureImage(capture, morphology, speciesId, captureApprovalTag) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/${id}`;
+      log.debug(
+        'approveCaptureImage',
+        capture,
+        morphology,
+        speciesId,
+        captureApprovalTag
+      );
 
-      log.debug(query);
+      // map legacy data to fields for new microservice
+      const newCapture = {
+        id: capture.uuid,
+        reference_id: capture.id,
+        session_id: capture.session_id, // no legacy equivalent
+        grower_account_id: capture.planterId,
+        planting_organization_id: capture.organization_id,
+        device_configuration_id: capture.device_configuration_id, // no legacy equivalent
+        image_url: capture.imageUrl,
+        lat: capture.lat,
+        lon: capture.lon,
+        gps_accuracy: capture.gps_accuracy,
+        captured_at: capture.timeCreated,
+        note: capture.note,
+        morphology,
+        species_id: speciesId, // need uuid
+        captureApprovalTag, // how does this fit into the new API?
+      };
 
-      return fetch(query, {
+      log.debug('newCapture data', newCapture);
+
+      // update the raw capture
+      fetch(`${FIELD_DATA_API}/raw-captures/${capture.uuid}`, {
         method: 'PATCH',
         headers: {
           'content-type': 'application/json',
           Authorization: session.token,
         },
         body: JSON.stringify({
-          id: id,
-          approved: true,
-          //revise, if click approved on a rejected pic, then, should set the pic approved, AND restore to ACTIVE = true
-          active: true,
-          morphology,
-          age,
-          captureApprovalTag,
-          speciesId: speciesId,
+          status: 'approved',
         }),
       }).then(handleResponse);
+
+      // add the new capture
+      fetch(`${TREETRACKER_API}/captures`, {
+        method: 'POST',
+        headers: {
+          'content-type': 'application/json',
+          Authorization: session.token,
+        },
+        body: JSON.stringify(newCapture),
+      }).then(handleResponse);
     } catch (error) {
       handleError(error);
     }
   },
-  rejectCaptureImage(id, rejectionReason) {
+  rejectCaptureImage(capture, rejection_reason) {
```

Comment:
Is there a reason the `rejection_reason` parameter is snake case?
It's best to keep use of camel case consistent within the Admin Panel and only convert when populating the API body (`rejection_reason: rejectionReason`).

Reply:
I had started naming the parameters to match the keys in the data before I got your message about preferring camelcase parameters.  Will switch.